core/src/test/scala/com.mediative.amadou/StageSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Mainly a scratchpad used while writing the stage code. Will be changed into unit tests.